cmake/Modules/FindClangFormat.cmake

```diff
@@ -0,0 +1,12 @@
+INCLUDE(FindPackageHandleStandardArgs)
+FIND_PROGRAM(CLANGFORMAT_EXECUTABLE clang-format HINTS ${CLANGFORMAT_ROOT})
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(clang-format DEFAULT_MSG CLANGFORMAT_EXECUTABLE)
+# only visible in advanced view
+MARK_AS_ADVANCED(CLANGFORMAT_EXECUTABLE)
+
+if (NOT CLANGFORMAT_EXECUTABLE)
+ set(CLANGFORMAT_FOUND FALSE)
+ message(WARNING "clang-format not found! You can't normalize codestyle by make target...")
```

Comment:
**#bug** Such message should be generated via user's script. It's can be not required. Or user can wish other error message. And you don't add target here (it's right).